File: app/src/main/java/com/casebeaumonde/notifications/response/RemoveNotificationResponse.java
package com.casebeaumonde.notifications.response;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class RemoveNotificationResponse {

@SerializedName("code")
@Expose
private Integer code;
@SerializedName("data")
@Expose
private Data data;

public Integer getCode() {
return code;
}

public void setCode(Integer code) {
this.code = code;
}

public Data getData() {
return data;
}

public void setData(Data data) {
this.data = data;
}

    public class Data {

        @SerializedName("message")
        @Expose
        private String message;
        @SerializedName("notification")
        @Expose
        private Notification notification;

        public String getMessage() {
            return message;
        }

        public void setMessage(String message) {
            this.message = message;
        }

        public Notification getNotification() {
            return notification;
        }

        public void setNotification(Notification notification) {
            this.notification = notification;
        }

        public class Notification {


        }
    }
}

File: app/src/main/java/com/casebeaumonde/activities/notifications/response/RemoveNotificationResponse.kt
package com.casebeaumonde.activities.notifications.response

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class RemoveNotificationResponse {

    @SerializedName("code")
    @Expose
    private var code: Int? = null

    @SerializedName("data")
    @Expose
    private var data: Data? = null

    fun getCode(): Int? {
        return code
    }

    fun setCode(code: Int?) {
        this.code = code
    }

    fun getData(): Data? {
        return data
    }

    fun setData(data: Data?) {
        this.data = data
    }

    class Data {
        @SerializedName("message")
        @Expose
        var message: String? = null

        @SerializedName("notification")
        @Expose
        var notification: Notification? = null

        inner class Notification
    }
}